textarea/textarea.go

```diff
@@ -111,14 +111,14 @@ type LineInfo struct {
 	CharOffset int
 }
 
-// Style that will be applied to the text area.
+// Styles that will be applied to the text area.
 //
-// Style can be applied to focused and unfocused states to change the styles
+// Styles can be applied to focused and unfocused states to change the styles
 // depending on the focus state.
 //
 // For an introduction to styling with Lip Gloss see:
 // https://github.com/charmbracelet/lipgloss
-type Style struct {
+type Styles struct {
```

Comment:
Any particular reason for the change here other than aesthetics?

Reply:
Consistency with other bubbles. The type alias should take care of backward compatibility.